views/option.erb

```diff
@@ -0,0 +1,14 @@
+
+<h1>Welcome <%= @player.name %></h1>
+
+<h3>Please choose and option:</h3>
+
+<form action='/play' method='post'>
```

Comment:
I really like this way of choosing the player move. Very neat and works well in practice